doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
```

Comment:
At some point in this file you should state clearly which releases ($releaserver) and architectures ($baearch) are supported. I guess that only CentOS/RHEL 7 and x86_64 ares supported.

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9